backend/src/main/java/com/wooteco/nolto/auth/ui/MemberArgumentResolver.java

```diff
@@ -27,6 +28,9 @@ public boolean supportsParameter(MethodParameter parameter) {
     @Override
     public User resolveArgument(MethodParameter parameter, ModelAndViewContainer mavContainer, NativeWebRequest webRequest, WebDataBinderFactory binderFactory) throws Exception {
         String credentials = AuthorizationExtractor.extract(Objects.requireNonNull(webRequest.getNativeRequest(HttpServletRequest.class)));
+        if (Objects.isNull(credentials)) {
```

Comment:
토큰이 공백인 경우는 어떻게 되나요??? 👀

Reply:
오 예리예리
엄 지금은 AuthService#validateToken에서 "유효하지 않은 토큰입니다."로 터집니다!
음음 헷갈리는게 있는데 공백으로만 이루어진 토큰이 존재하지 않는 것인지 토큰인지 유효하지 않는 것인지?? 가 애매하네유유
존재하지 않는 것으로 보면 아래와 같이 바꾸겠슴당~
``
```suggestion
        if (Objects.isNull(credentials) || credentials.trim().isEmpty()) {
```